Cargo.lock

```diff
@@ -2342,6 +2357,12 @@ version = "0.3.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "624a8340c38c1b80fd549087862da4ba43e08858af025b236e509b6649fc13d5"
 
+[[package]]
+name = "openssl-probe"
+version = "0.1.4"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "28988d872ab76095a6e6ac88d99b54fd267702734fd7ffe610ca27f533ddb95a"
+
```

Comment:
Yikes 😬  it comes from `rustls-native-certs` and it doesn't seem it could be in any way skipped (no feature flags in that crate). I looked up that crate's code and it doesn't do any interaction with OpenSSL so I guess we can live with that.

Reply:
Interesting, I think it does say something about it [in the docs](https://crates.io/crates/rustls-native-certs) now that you mention it:

> On Linux and other UNIX-like operating systems, the openssl-probe crate is used to discover the filename of the system CA bundle.